### ISSUE TITLE
Add the missing mouse hover focus indicators to Submit an App page

### ISSUE THREAD

**Issue by ctrance** [ucla-oarc-mobile/casa-on-rails#208]
@krozett The following are missing mouse hover focus indicators on Submit an App page
a.	apps@oit.ucla.edu link
![image](https://user-images.githubusercontent.com/1034174/108111989-591e9780-704a-11eb-86e8-df952fb49daa.png)
b.	Submit App button
![image](https://user-images.githubusercontent.com/1034174/108112005-5e7be200-704a-11eb-862b-9a23edfc5769.png)
c.	Privacy policy template generator
![image](https://user-images.githubusercontent.com/1034174/108112021-63409600-704a-11eb-8cfd-7899b5068e16.png)
d.	Remove and add buttons for Authors and Organizations
![image](https://user-images.githubusercontent.com/1034174/108112039-69367700-704a-11eb-9537-a8a65cf7d5a6.png)
e.	Categories list items
![image](https://user-images.githubusercontent.com/1034174/108112048-6dfb2b00-704a-11eb-93f0-69c0e4539552.png)

Accessibility success criterion:
* https://www.w3.org/TR/WCAG21/#use-of-color
* https://www.w3.org/TR/WCAG21/#non-text-contrast

**Comment by ctrance** [comment]
Closing old tickets that we will not fix for now.
